setup.py

```diff
@@ -810,6 +909,7 @@ def summary_report(self, feature):
             (feature.imagequant, "LIBIMAGEQUANT"),
             (feature.tiff, "LIBTIFF"),
             (feature.freetype, "FREETYPE2"),
+            (feature.raqm, "RAQM (Text shaping)"),  # TODO!!!
```

Comment:
What is to be done here?

Reply:
I wanted to add info about the selected configuration. I pushed it now.